doc/source/ray-core/examples/plot_pong_example.ipynb

```diff
@@ -31,7 +31,7 @@
     "To run the application, first install some dependencies.\n",
     "\n",
     "```bash\n",
-    "pip install gym[atari]\n",
+    "pip install gymnasium[atari] gym==0.26.2\n",
```

Comment:
why both?

Reply:
Thanks for woking on this issue, having RLlib supporting gymnasium is something I have been waiting for. Do we know roughly when you are planning to release version 2.3?